PRINCIPLES.md

```diff
@@ -55,7 +55,7 @@ _Principle: Great projects already have many ingredients to succeed. First: do n
 Identify projects that have a real shot at being a useful tool in the evolving box of cloud native technology. This is a mix of mature and early-stage projects.   Early stage may not have all the criteria we want: diverse contributor bases, formalized governance, interoperability, cloud-native designs, quality bar, etc. 
 
 Some considerations:
-- Transparent, consistent technical and governance quality bar for [graduation](https://www.cncf.io/projects/graduation-criteria/) from incubation
+- Transparent, consistent technical and governance quality bar for [graduation](/process/graduation_criteria.adoc) from incubation
```

Comment:
This LGTM. I am not a web indvidiual - I program infratructure. When working with documentation I have  been told to always use a trialing slash is it prevents a dual hit to a website. A quick google search turns up this recommendation:
https://www.cloudways.com/blog/trailing-slashes-urls/

Essentially use a trailing slash on directories, or not if it is definitely a file.

This change LGTM.